mobile/src/main/res/values-es/strings.xml

```diff
@@ -416,8 +416,8 @@ conectarse a un servidor</string>
   <string name="Vault_CreateFolder_SheetAction">Crear una nueva carpeta</string>
   <string name="ServerSettings_ServerName_ToolTip">Dale un nombre a tu servidor para reconocerlo si agregas más servidores más adelante. Podría ser el nombre de la organización, el nombre del proyecto, etc.</string>
   <plurals name="Vault_Importing_SheetProgress">
-    <item quantity="one">%s archivo importado</item>
```

Comment:
Why removing the %s you can assign it directly from the code.

Reply:
because the progress percent is in mutable livedata and been observed in the bottomSheet class so I can not set it in the activity where i get the string Vault_Importing_SheetProgress